src/components/Footer/Footer.scss

```diff
@@ -0,0 +1,117 @@
+@import '/src/prepare-styles/colors.scss';
+@import '/src/prepare-styles/mixins.scss'; 
+@import '/src/prepare-styles/usable-fonts.scss'; 
+
+$tablet-min-width: 640px;
```

Comment:
Please, move this constants to new created separate file(prepare-styles/constant.scss) for example))